Packs/ExceedLMS/Integrations/IAMExceedLMS/IAMExceedLMS.py

```diff
@@ -0,0 +1,355 @@
+
+        user_data = {
+            'is_active': 'false'
+        }
+        return self.update_user(user_id, user_data)
+
+    def get_app_fields(self) -> Dict[str, Any]:
+        """ Gets a dictionary of the user schema fields in the application and their description.
+
+        :return: The user schema fields dictionary
+        :rtype: ``Dict[str, str]``
+        """
+        uri = '/api/v2/users.json/'
+        params = {'api_key': self.api_key}
+        res = self._http_request(
+            method='GET',
+            url_suffix=uri,
+            params=params
+        )
+        if isinstance(res, dict):
+            res = [res]
+        if len(res) > 0:
+            res = res[0]
+        return {key: "" for key, val in res.items()}
+
+    @staticmethod
+    def handle_exception(user_profile: IAMUserProfile,
+                         e: Union[DemistoException, Exception],
+                         action: IAMActions):
+        """ Handles failed responses from the application API by setting the User Profile object with the result.
+            The result entity should contain the following data:
+            1. action        (``IAMActions``)       The failed action                       Required
+            2. success       (``bool``)             The success status                      Optional (by default, True)
+            3. skip          (``bool``)             Whether or not the command was skipped  Optional (by default, False)
+            3. skip_reason   (``str``)              Skip reason                             Optional (by default, None)
+            4. error_code    (``Union[str, int]``)  HTTP error code                         Optional (by default, None)
+            5. error_message (``str``)              The error description                   Optional (by default, None)
+
+            Note: This is the place to determine how to handle specific edge cases from the API, e.g.,
+            when a DISABLE action was made on a user which is already disabled and therefore we can't
+            perform another DISABLE action.
+
+        :type user_profile: ``IAMUserProfile``
+        :param user_profile: The user profile object
+
+        :type e: ``Union[DemistoException, Exception]``
+        :param e: The exception object - if type is DemistoException, holds the response json object (`res` attribute)
+
+        :type action: ``IAMActions``
+        :param action: An enum represents the current action (GET, UPDATE, CREATE, DISABLE or ENABLE)
+        """
+        if isinstance(e, DemistoException) and e.res is not None:
+            error_code = e.res.status_code
+
+            if action == IAMActions.DISABLE_USER and error_code in ERROR_CODES_TO_SKIP:
+                skip_message = 'Users is already disabled or does not exist in the system.'
+                user_profile.set_result(action=action,
+                                        skip=True,
+                                        skip_reason=skip_message)
```

Comment:
If I understand correctly, this is not really used - if so, you can remove this part

Reply:
Done